docs/advanced/advanced.md

```diff
@@ -237,10 +238,40 @@ INFO[2019-01-22T15:16:20.541027133-07:00] Wrote kubeconfig /etc/rancher/k3s/k3s.
 INFO[2019-01-22T15:16:20.541049100-07:00] Run: k3s kubectl                             
 ```
 
-The output will likely be much longer as the agent will create a lot of logs. By default the server
+The output will likely be much longer as the agent will create a lot of logs. By default, the server
 will register itself as a node (run the agent).
 
-## Additional preparation for (Red Hat/CentOS) Enterprise Linux
+## Additional OS Preparations
+
+### Additional preparation for Debian "buster" based distributions
+
+Several popular OSes based on Debian "buster" ship a version of iptables, between v1.8.0-v1.8.4. These versions contain a bug which cause the accumulation of too many rules which will bog down and crash the K3s server. See [this issue](https://github.com/k3s-io/k3s/issues/3117) for more background. 
```

Comment:
```suggestion
Several popular Linux distributions based on Debian "buster" ship a version of IPTables between v1.8.0-v1.8.4. These versions contain a bug which causes the accumulation of duplicate rules, which negatively affects the performance and stability of the node. See [Issue #3117](https://github.com/k3s-io/k3s/issues/3117) for more background. 
```

Reply:
It appears that iptables is never capitalized in any of the documentation I could find online. Every other change was added.